src/rwa003/mgr-rpc.t.sol

```diff
@@ -0,0 +1,125 @@
+pragma solidity 0.5.12;
+
+import {DssSpellTestBase} from "./spell-rpc.t.sol";
+import "dss-interfaces/Interfaces.sol";
+import {TinlakeManager} from "../mgr.sol";
+
+interface EpochCoordinatorLike {
+    function closeEpoch() external;
+    function currentEpoch() external returns(uint);
+}
+
+interface Hevm {
+    function warp(uint) external;
+    function store(address,bytes32,bytes32) external;
+    function load(address,bytes32) external returns (bytes32);
+}
+
+interface Root {
+    function relyContract(address, address) external;
+}
+
+interface MemberList {
+    function updateMember(address, uint) external;
+}
+
+interface AssessorLike {
+    function calcSeniorTokenPrice() external returns (uint);
+}
+
+interface FileLike {
+    function file(bytes32 what, address data) external;
+}
+
+interface ERC20Like {
+    function mint(address, uint256) external;
+}
+
+contract KovanRPC is DssSpellTestBase {
+    DSTokenAbstract  public drop;
+    TinlakeManager dropMgr;
+
+    // Tinlake
+    Root constant root = Root(ROOT);
+    MemberList constant memberlist = MemberList(MEMBERLIST);
+    EpochCoordinatorLike constant coordinator = EpochCoordinatorLike(COORDINATOR);
+
+    address self = address(this);
+
+    function setUp() public {
+        super.setUp();
+        self = address(this);
+        dropMgr = TinlakeManager(address(mgr));
+        drop = DSTokenAbstract(address(dropMgr.gem()));
+
+        // welcome to hevm KYC
+        hevm.store(address(root), keccak256(abi.encode(address(this), uint(0))), bytes32(uint(1)));
+
+        root.relyContract(address(memberlist), address(this));
+
+        memberlist.updateMember(self, uint(-1));
+        memberlist.updateMember(address(dropMgr), uint(-1));
+
+        // set this contract as ward on the mgr
+        hevm.store(address(dropMgr), keccak256(abi.encode(self, uint(0))), bytes32(uint(1)));
+        assertEq(dropMgr.wards(self), 1);
+
+        // file MIP21 contracts 
+        FileLike(address(mgr)).file("liq", LIQ);
```

Comment:
If I understand it correctly it should not be an issue here, as we are filing some values for testing in the tests & not in the spell itself

Reply:
Yes, this is a test and probably low consequence.  I was being lazy leaving my comment here because I was short on time.  I'll look in a sec, but my comment here is more that the `MGR` would have been the wrong address somewhere in the code.  This should be detectable and we should have a simple test for it somewhere, if not, we probably aren't testing the values in these contracts well enough after deploy.

Reply:
aaah yes, I see what you mean. We are testing the values here. lib/blob/3cba609ef9c0ec2d8a09039377ee2b31c9aab90e/src/rwa003/spell-rpc.t.sol#L647
The reason why the tests did not fail is that the spell is actually pulling the LIQ oracle info directly form the chainlog https://github.com/centrifuge/tinlake-maker-lib/blob/3cba609ef9c0ec2d8a09039377ee2b31c9aab90e/src/rwa003/spell.sol#L60

Reply:
I like that explanation. 👍